docs/source/options.rst

```diff
@@ -0,0 +1,107 @@
+.. Copyright 2016 The Cartographer Authors
+
+.. Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+..      http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+=======================
+Configuration Reference
+=======================
+
+Top-level options of the Cartographer ROS integration:
+
+map_frame
+  The ROS frame id to use for publishing submaps, the parent frame of poses,
+  usually "map".
+
+tracking_frame
+  The ROS frame id of the frame that is tracked by the SLAM algorithm. If an IMU
+  is used it should be at its position, although it might be rotated. A common
+  choice is "base_link".
+
+published_frame
+  The ROS frame id to use as the child frame for publishing poses. For example
+  "odom" if an "odom" frame is supplied by a different part of the system. In
+  this case the pose of "odom" in the map frame is to be published.
```

Comment:
... will be published.

Add a second example for when odom is not published by another node.